tests/test_integration.py

```diff
@@ -1,39 +1,50 @@
+import pytest
 from pathlib import Path
 
 import asdf
 import yaml
 
 
-def test_resources():
+def get_resources():
     resources_root = Path(__file__).parent.parent / "resources"
+
+    return list(resources_root.glob("**/*.yaml"))
+
+
+@pytest.mark.parametrize("resource_path", get_resources())
```

Comment:
It's nice to add an `ids` argument here so that instead of this:

```
tests/test_integration.py::test_resource[resource_path0] PASSED
```
we get something like this:
```
tests/test_integration.py::test_resource[resources/asdf-format.org/manifests/transform-1.5.0.yaml] PASSED
```

Reply:
See 4a76d9d.